4/run-silo-local.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+export ADVERTISEDIP=`ifconfig | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*' | grep -Eo '([0-9]*\.){3}[0-9]*' | grep -v '127.0.0.1'`
+export PRIMARYADDRESS=`ifconfig | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*' | grep -Eo '([0-9]*\.){3}[0-9]*' | grep -v '127.0.0.1'`
+export GATEWAYPORT=3002
+export SILOPORT=2002
```

Comment:
Let's leave those ports to be the default ...001 so that whoever runs those scripts, cannot really distinguish between a silo running locally and a silo running locally.